tools/cli/commands/install.js

```diff
@@ -105,6 +105,10 @@ export async function handler( argv ) {
 				execa( 'composer', await getInstallArgs( project, 'composer', argv ), {
 					cwd: projectDir( project ),
 					stdio,
+				} ).catch( err => {
+					err.fix =
+						'To resolve issues, try running the command: `cd projects/plugins/jetpack && composer install`';
```

Comment:
Instead of running `composer install` yourself, maybe we should suggest running `jetpack install xxx` just for the element that's causing issues?

Reply:
The problem is that there is a variety of solutions and the current script is hiding composer error messages.
There is usually a message to run "composer update", but some of the times it wants to downgrade or be run with --ignore-platform-deps, depending on your local setup.

This is meant to at least point you to a right direction, not solve every conceivable problem.
I think in general these scripts try to do too much and are not humble enough to admit when they can't do things for you

Reply:
I like @jeherve's suggestion, although I'd recommend including `-v` to get the verbose output since, unless it was a transient network error or something, it'll probably fail again.

BTW: Running `composer update` on a project where composer.lock is checked in could cause you problems when submitting the PR, as it might update indirect deps like changelogger's dep on various Symfony packages. Our `tools/composer-update-monorepo.sh` script is safer as it avoids updating those. If you need `--ignore-platform-deps`, that probably means your dev environment is screwed up somehow (e.g. a too-old version of PHP).